py3status/module.py

```diff
@@ -479,6 +472,20 @@ def process_composite(self, response):
                 else:
                     item[key] = value
 
+            # remove urgent if not allowed
+            if not self.allow_urgent:
+                if "urgent" in item:
+                    del item["urgent"]
+            # if urgent we want to set this to all parts
+            elif urgent:
+                if self.urgent_background:
+                    item["background"] = self.urgent_background
+                    item["color"] = "#FFFFFF"
+                    if "urgent" in item:
+                        del item["urgent"]
```

Comment:
Can you explain why delete "urgent" key from item here?

Reply:
Some items have `urgent` there. You can try commenting it out. i3 have a way for users to specify `urgent` background on i3bar... except that it will be applied to all urgent modules and no way to customize colors. https://i3wm.org/docs/userguide.html#_colors
```python
bar {
    colors {
        background #000000
        statusline #ffffff
        separator #666666

        focused_workspace  #4c7899 #285577 #ffffff
        active_workspace   #333333 #5f676a #ffffff
        inactive_workspace #333333 #222222 #888888
        urgent_workspace   #2f343a #900000 #ffffff
        binding_mode       #2f343a #900000 #ffffff
    }
}
```
In this PR, I delete `urgent` keys before we send it to i3bar... so i3bar won't activate their i3bar urgent color... and replacing `urgent` with user-defined background and white foreground.... 

Only if `urgent_background` is defined.

Reply:
Aaaah, I see, makes sense.

Reply:
So you're basically hiding something from i3bar and diverging from the default i3 behavior here right?

Reply:
If `urgent_background` is defined... It will delete all urgent and replace it with custom fake urgent background. Otherwise, it does not matter what background color (or any other color options) users have... because i3bar would replace it with its `urgent` color.

Otherwise, it should proceed as normal (real) urgent behavior.